### PR TITLE
feat: add rabbitmq sample apps

### PR DESCRIPTION
**Pull request by wconti27**
## Motivation

Add RabbitMQ Endpoints for nodejs and python

## Changes

More testing!

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] [Relevant label](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md) (`run-parametric-scenario`, `run-profiling-scenario`...) are presents
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
    * [ ] To R&P team: locally build and push the image to hub.docker.com 
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
    * [ ] Once merged, add (or remove) it in system-test-dasboard nightly

### CODE REVIEW

File: tests/integrations/test_dsm.py [hunk 1]
```diff
@@ -182,15 +190,16 @@ class Test_DsmSQS:
     """ Verify DSM stats points for AWS Sqs Service """
 
     def setup_dsm_sqs(self):
-        self.r = weblog.get("/dsm?integration=sqs")
+        self.r = weblog.get("/dsm?integration=sqs&timeout=60", timeout=61)
 
-    @bug(weblog_variant="flask-poc", reason="DSM checkpoints for AWS SQS from dd-trace-py are not being receieved.")
+    @bug(weblog_variant="flask-poc", reason="DSM checkpoints for AWS SQS from dd-trace-py are not being received.")
+    @bug(weblog_variant="express4", reason="DSM checkpoints for AWS SQS from dd-trace-js are not being received.")
```
Comment (robertomonteromiguel) [inline tests/integrations/test_dsm.py:196]: is there a jira issue related with this bug?

File: tests/integrations/test_dsm.py [hunk 2]
```diff
@@ -66,16 +66,24 @@ def setup_dsm_rabbitmq(self):
     def test_dsm_rabbitmq(self):
         assert self.r.text == "ok"
 
+        if context.library == "nodejs":
```
Comment (robertomonteromiguel) [inline tests/integrations/test_dsm.py:69]: could we add a comment about where these hashes come from and why nodejs is different?

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]

File: tests/integrations/test_dsm.py (hunk 2 above)
Comment (wconti27) [reply tests/integrations/test_dsm.py:73]: Added!

File: tests/integrations/test_dsm.py (hunk 1 above)
Comment (wconti27) [reply tests/integrations/test_dsm.py:200]: I'm not sure, I will be looking into this issue after and will either fix it with another PR or create an issue if I can't find the cause

Comment (robertomonteromiguel) [reply tests/integrations/test_dsm.py:200]: ok

**Review by robertomonteromiguel — APPROVED** [review]

**Pull request merged**